File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/vip_pack_group_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class VipPackGroupVO:
    """VIP套餐分组响应"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    name: str = None
    description: str = None
    sort_weight: int = None
    packs: List[VipPackVO] = None

File: sdkwork-app-sdk-go/types/vip_pack_group_vo.go
package types

// VIP套餐分组响应
type VipPackGroupVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Name string `json:"name"`
	Description string `json:"description"`
	SortWeight int `json:"sortWeight"`
	Packs []VipPackVO `json:"packs"`
}
